service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedEventControllerTest.java

```diff
@@ -184,26 +208,27 @@ void shouldNotSendEmailNotificationWhenNoChangesToOrder() throws NotificationCli
                 eq(NOTICE_OF_PLACEMENT_ORDER_UPLOADED_TEMPLATE),
                 eq("local-authority@local-authority.com"),
                 eq(expectedParameters()),
-                eq("1"));
+                eq(CASE_ID));
 
             verify(notificationClient, never()).sendEmail(
                 eq(NOTICE_OF_PLACEMENT_ORDER_UPLOADED_TEMPLATE),
                 eq("representative@example.com"),
                 eq(expectedParameters()),
-                eq("1"));
+                eq(CASE_ID));
         }
 
         private Map<String, Object> expectedParameters() {
             return Map.of(
-                "respondentLastName", "Nelson",
-                "caseUrl", String.format("%s/case/%s/%s/%s", "http://fake-url", JURISDICTION, CASE_TYPE, 1L));
+                "respondentLastName", "Jones",
```

Comment:
err why Jones -> Nelson?

Reply:
NotifyAdminOrderIssuedTestHelper expects respondentLastName to be Jones, and since it is used in 3 controller tests I've changed test data in controllers where it wasn't consistent (so that I can cleanly utilise my test helper)

Reply:
justification is that 3 controllers trigger the same notification so it made sense to centralise the code used for testing it, so that I can call it from 3 different places rather than duplicate it 3 times